kolibri/core/logger/serializers.py

```diff
@@ -58,8 +62,19 @@ class Meta:
                   'end_timestamp', 'completion_timestamp', 'mastery_criterion', 'mastery_level', 'complete')
 
     def get_pastattempts(self, obj):
-        # will return a list of the latest 10 correct and hint_taken fields for each attempt.
-        return AttemptLog.objects.filter(masterylog__summarylog=obj.summarylog).values('correct', 'hinted', 'error').order_by('-start_timestamp')[:10]
+        mastery_criterion = json.loads(obj.mastery_criterion)
```

Comment:
@rtibbles @jamalex I'm getting an type error here since `obj.mastery_criterion` is already a dictionary. Should this just be

```diff
- mastery_criterion = json.loads(obj.mastery_criterion)
+ mastery_criterion = obj.mastery_criterion
```

Reply:
Yeah, that seems right, as it is a JSON field and should be deserialized already on access.